### PR TITLE
[fastboot] fastboot enhancement: Use warm-boot infrastructure for fast-boot

### PR DESCRIPTION
**Pull request by arfeigin**
This PR should be merged together with the sonic-utilities PR (https://github.com/sonic-net/sonic-utilities/pull/2286) and sonic-sairedis PR (https://github.com/sonic-net/sonic-sairedis/pull/1100).

Use redis contents from dump file in fast-reboot.

#### Why I did it
Improve fast-reboot flow by utilizing the warm-reboot infrastructure.
This followes https://github.com/sonic-net/SONiC/blob/master/doc/fast-reboot/Fast-reboot_Flow_Improvements_HLD.md
#### How I did it
Use redis contents from dump file in fast-reboot as it is being used in warm-reboot
#### How to verify it
Community test for fast-reboot
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [X] 202205

### CODE REVIEW

**Review by vaibhavhd — APPROVED** [review]
Please check PR failures. Also it may be a better idea to not merge this until sairedis and utilities PR are merged too. So that in this PR submodule can also be advanced for a complete feature check-in to buildimage repo.

**Comment by arfeigin** [comment]
Checkers fail due to kvmtest-multi-asic-t1-lag tests fail because they exceed the timeout of 360m. Looks like the timeout needs to be increased.
I see similar issue also in https://github.com/sonic-net/sonic-buildimage/pull/11754 and similar issues reported with other kvm tests: https://github.com/sonic-net/sonic-buildimage/pull/10510, https://github.com/sonic-net/sonic-buildimage/pull/11593, https://github.com/sonic-net/sonic-buildimage/pull/11546.

As well as @vaibhavhd noted it would be better to first merge the sonic-utilities PR (https://github.com/sonic-net/sonic-utilities/pull/2286) and sonic-sairedis PR (https://github.com/sonic-net/sonic-sairedis/pull/1100).

**Comment by liat-grozovik** [comment]
@vaibhavhd could you please help to merge?

**Pull request merged**
